network-api/networkapi/wagtailpages/models.py

```diff
@@ -738,13 +738,15 @@ def entries_by_tag(self, request, tag, *args, **kwargs):
 
     # helper function to resolve category slugs to actual objects
     def get_category_object_for_slug(self, category_slug):
-        category_object = None
 
         # We can't use .filter for @property fields,
         # so we have to run through all categories =(
         for bpc in BlogPageCategory.objects.all():
             if bpc.slug == category_slug:
                 category_object = bpc
+                break
+        else:
```

Comment:
cool, I didn't know this existed

Reply:
for-else FTW! won't forget this handy little language feature.

Reply:
I learned about this last week, and now I see it everywhere :D